src/components/target/index.js

```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react';
+
+import { Marker, useMapEvents } from 'react-leaflet';
+import myIcon from '../Map/Icon';
+
+const AddMarkerToClick = props => {
+  const map = useMapEvents({
+    click(e) {
+      props.targets.push(e.latlng);
+      props.sendLatLng(e.latlng);
+    },
+  });
+  return (
+    <>
+      {props.targets.map((target, index) => (
+        <Marker position={target} icon={myIcon} key={index}></Marker>
```

Comment:
If you are not passing any children to a component, you can self close it
```suggestion
        <Marker position={target} icon={myIcon} key={index} />
```